subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa_kmu.h

```diff
@@ -26,10 +26,14 @@
 /* Retrieve KMU slot number for PSA key id. */
 #define CRACEN_PSA_GET_KMU_SLOT(key_id) ((key_id)&0xff)
 
-#define CRACEN_KMU_KEY_USAGE_SCHEME_PROTECTED 0
-#define CRACEN_KMU_KEY_USAGE_SCHEME_SEED      1
-#define CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED 2
-#define CRACEN_KMU_KEY_USAGE_SCHEME_RAW	      3
+#define CRACEN_KMU_KEY_USAGE_SCHEME_PROTECTED 0x00
+#define CRACEN_KMU_KEY_USAGE_SCHEME_SEED      0x01
+#define CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED 0x02
+#define CRACEN_KMU_KEY_USAGE_SCHEME_RAW	      0x03
+#define CRACEN_KMU_KEY_USAGE_SIGN_MESSAGE     0x04
```

Comment:
I don't think you use these? Metadata have sign/verify info. Possibly we can just use "RAW" for this, or add a direct-to-pke-ram if you add that.

Reply:
Ah yeah added this in the beginning but is not use gonna revert